scram/route_manager/models.py

```diff
@@ -98,45 +101,48 @@ class Entry(models.Model):
     expiration_reason = models.CharField(
         help_text="Optional reason for the expiration",
         max_length=200,
-        null=True,
         blank=True,
+        default="",
     )
 
-    def delete(self, *args, **kwargs):
-        """Set inactive instead of deleting, as we want to ensure a history of entries."""
-        if not self.is_active:
-            # We've already expired this route, don't send another message
-            return
-        else:
-            # We don't actually delete records; we set them to inactive and then tell the translator to remove them
-            logging.info(f"Deactivating {self.route}")
-            self.is_active = False
-            self.save()
-
-            # Unblock it
-            async_to_sync(channel_layer.group_send)(
-                f"translator_{self.actiontype}",
-                {
-                    "type": "translator_remove",
-                    "message": {"route": str(self.route)},
-                },
-            )
-
     class Meta:
         """Ensure that multiple routes can be added as long as they have different action types."""
 
         unique_together = ["route", "actiontype"]
         verbose_name_plural = "Entries"
 
     def __str__(self):
-        """Summarize the most important fields to something easily readable."""
+        """Summarize the most important fields to something easily readable."""  # noqa: DOC201
         desc = f"{self.route} ({self.actiontype})"
         if not self.is_active:
             desc += " (inactive)"
         return desc
 
+    def delete(self, *args, **kwargs):
```

Comment:
Why move this around?

Reply:
The Django checks enforce an order. Custom functions, then Meta class, then stuff like `__str__`, etc.